### ISSUE TITLE
Base64 encoding raises error when using Python3

### ISSUE THREAD

**Issue by johngian** [mozilla/mozilla-django-oidc#89]
When ``OIDC_RP_CLIENT_SECRET_ENCODED==False`` authentication raises the following error on callback:

```
Exception Type: TypeError at /oidc/callback/
Exception Value: a bytes-like object is required, not 'str'
```

This gets triggered on python 3.x only.

**Comment by willkg** [comment]
Did this issue cover the problem I fixed in #91?

**Comment by johngian** [comment]
No, the issue is in this line:
https://github.com/mozilla/mozilla-django-oidc/blob/master/mozilla_django_oidc/auth.py#L85
